ramls/loan.json

```diff
@@ -242,6 +242,11 @@
       },
       "additionalProperties": false
     },
+    "itemEffectiveLocationIdAtCheckOut": {
```

Comment:
Could you please update API versions where the loan schema is used.

Reply:
Updated circulation.raml 9.2 -> 9.3